pkg/core/database/interfaces.go

```diff
@@ -95,8 +95,11 @@ type Transaction interface {
 	FetchOutputUnlockHeight(destkey []byte) (uint64, error)
 
 	// StoreBidValues stores the D and K values passed by the caller in
-	// the database.
-	StoreBidValues([]byte, []byte) error
+	// the database, as well as the expiry height. It should be passed
+	// the transaction locktime as a third argument, as the database
+	// can infer the current height and consequently, the expiry height,
+	// on its own.
+	StoreBidValues([]byte, []byte, uint64) error
 
 	// FetchBidValues retrieves the last stored D and K values from
```

Comment:
FetchBidValues now returns `bid values with the lowest height` but not the last stored, right ?